pages/githubandmarkdown.md

```diff
@@ -67,18 +67,21 @@ Please, make sure to ALWAYS work on your own personal account via forking and pu
 
 
 ###Create a new file as your personal MD Wiki page and commit your changes
-Now, you can edit the files in your repo and the changes you make will be reflected on your &lt;username&gt;.github.io website. To edit, click on the `pages` folder. Then, click `New File` and name the file exactly the same as your GitHub username. We created pages for you using your GitHub username, so it is important that you stick with that name. For reference, an example of a link to a personal page is EmilyLarkin.github.io/pages/EmilyLarkin.md, so yours should be named in the same manner, only using your username.
-
-In the new file, please type your name, your location/time zone, and OS (including version). In addition, make sure to add a short description of yourself as well, so your fellow virtual interns know who you are. Also try to use several Markdown elements in your page to get an idea for how it works. Then, scroll down to the bottom of the page, select the ` Create a new branch for this commit and start a pull request. Learn more about pull requests.` radio box and input `arbitrary name`. You can see the example below.
+ Before editing the file ` Create a new branch `. You can see the example below.
 
 ![New Branch](uploads/images/newbranch.png)
 
+Now, you can edit the files in your repo and the changes you make will be reflected on your &lt;username&gt;.github.io website. To edit, click on the `pages` folder. Then, click `New File` and name the file exactly the same as your GitHub username. We created pages for you using your GitHub username, so it is important that you stick with that name. For reference, an example of a link to a personal page is EmilyLarkin.github.io/pages/EmilyLarkin.md, so yours should be named in the same manner, only using your username.
```

Comment:
>Now, you can edit the files in your repo and the changes you make will be reflected on your &lt;username&gt;.github.io website.

Since its on the branch not master the changes won't actually be reflected on their github pages. That is why we make them use the rawgit link in the next paragraph. I would change it to:


> ~~Now, you can edit the files in your repo and the changes you make will be reflected on your &lt;username&gt;.github.io website.~~ To create your page click on the `pages` folder. Then, click `New File` and name the file exactly the same as your GitHub username.